python/fio/fio_plugin.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import typing
+import json
+import subprocess
+from traceback import format_exc
+from typing import Union
+from pathlib import Path
+
+from arcaflow_plugin_sdk import plugin
+from fio_schema import (
+    FioJob,
+    FioSuccessOutput,
+    FioErrorOutput,
+    fio_output_schema,
+)
+
+
+@plugin.step(
+    id="workload",
+    name="fio workload",
+    description="run an fio workload",
+    outputs={"success": FioSuccessOutput, "error": FioErrorOutput},
+)
+def run(
+    params: FioJob,
+) -> typing.Tuple[str, Union[FioSuccessOutput, FioErrorOutput]]:
+    outfile_temp_path = Path("fio-plus.json")
+    infile_temp_path = Path("fio-input-tmp.fio")
```

Comment:
Your temp files are getting left behind as remnants, even when simply running the `test_fio_plugin.py` script. These should get cleaned up on exit.

Reply:
Even with `cleanup: true`?

Reply:
I see now -- The files are explicitly left behind by the `test_fio_plugin.py` script as it sets `cleanup: false`. Is this intended as part of the test?

Reply:
Yes, but I mostly got lazy about cleaning them up since I expected folks to run it in the container. I added code to clean up these files.